content/reference/admin/private_locations/private_packages/index.md

```diff
@@ -117,6 +117,30 @@ This configuration includes the following parameters:
 - **bucket**: The name of the bucket where packages are uploaded to on GCP Cloud Storage.
 - **path:** The path of a folder in Cloud Storage bucket. (optional)
 
+#### Filesystem Storage
+This option allows the storage of simulations directly on the control-plane filesystem.
+```bash
+  repository = {
+    type = "filesystem"
+    directory = "/data/gatling-repository"
+    upload {
+      directory = "/tmp" # (optional, default: /tmp)
```

Comment:
Also add your warning about directories in comments so that the user will have it after his glorious example copy-paste :)